drawBot/drawBotDrawingTools.py

```diff
@@ -363,7 +363,7 @@ def pages(self):
                     break
         return tuple(DrawBotPage(instructionSet) for instructionSet in instructions)
 
-    def saveImage(self, paths, multipage=None):
+    def saveImage(self, paths, multipage=None, **options):
```

Comment:
would it be an idea to move `multipage` to options too?
I understand this is done to keep scripts with `saveImage(path, True)` working
but multipage feels like option :) where a pdf and image context are interested in...

Reply:
Yeah, let's not break the api, but it is indeed a context-specific option. I'll see how to unify this.

Together with docs, it would be nice if we could validate the option names. Right now any misspelling is simply ignored.

Reply:
`*args` could just capturing `multipage` and warn and put it inside the options 
`def saveImage(self, paths, *deprecatedArgs, **options)`

related to validation and docs: each context could have a `ctx.saveImageOptions` a list of all possible options (similar as `fileExtensions`)

Reply:
`ctx.saveImageOptions` now exists, and is a list of `(optionName, optionDoc)` tuples. Validation is done in the public `saveImage()` entry point. @typemytype can you see how this can be integrated into the docs?

Reply:
I left the public `saveImage()` function signature as it is (only adding `**options`). I prefer that over adding a deprecation warning. I'm fine with people using `multipage` as a positional argument, let's just not do it in our examples.